code/game/objects/items/weapons/twohanded.dm

```diff
@@ -1034,4 +1034,82 @@
 	cart.mybroom = src
 	cart.put_in_cart(src, user)
 
+/obj/item/twohanded/push_broom/traitor
+	name = "titanium push broom"
+	desc = "This is my BROOMSTICK! All of the functionality of a normal broom, but at least half again more robust."
+	attack_verb = list("smashed", "slammed", "whacked", "thwacked", "swept")
+	force = 10
+	force_unwielded = 10
+	force_wielded = 25
+
+/obj/item/twohanded/push_broom/traitor/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/parry, _stamina_constant = 2, _stamina_coefficient = 0.5, _parryable_attack_types = NON_PROJECTILE_ATTACKS)
+
+/obj/item/twohanded/push_broom/traitor/examine(mob/user)
+	. = ..()
+	if(isAntag(user))
+		. += "<span class='warning'>When wielded, the broom has different effects depending on your intent, similar to a martial art. \
+		      Help intent will sweep foes away from you, disarm intent will sweep their legs from under them, grab intent will confuse \
+			  and minorly fatigue them, and harm intent will hit them normally.</span>"
```

Comment:
Could be written in another way to avoid "will" repetition.
Could maybe use a list? Dont know if it would take too much space on the chat.
"Help - pushes people away"
"Disarm - sweeps your target legs from under them"